### ISSUE TITLE
Sorting of policy names in Readme

### ISSUE THREAD

**Issue by Speravir** [mozilla/policy-templates#837]
Please, sort on next update the policy names in [README.md](https://github.com/mozilla/policy-templates/blob/master/README.md) alphabetically. As far as I see these are out of order at time of writing:
* _DisableFirefoxAccounts_ (one up, i.e. switch with the other setting)
* _DisplayMenuBar_ (one up)
* _DefaultDownloadDirectory_ (27 up!)
* _FirefoxHome_ (one up)
* _Handlers_ (one up)
* _PrimaryPassword_ (14 down!)
* _SearchEngines -> Add_ (three up)

I actually noticed this because I first did not find _DefaultDownloadDirectory_ which had been mentioned elsewhere.

**Comment by mkaply** [comment]
Yeah, sorry. PrimaryPassword used to be MasterPassword and I didn't move.

I'll fix this in today's release.

**Comment by mkaply** [comment]
Can you double check my work? :)

**Comment by Speravir** [comment]
> Can you double check my work? :)

Of course, I can. :-)

All but one are fine, was this intentional?
> * _SearchEngines -> Add_ (three up)

Another pettiness I originally wanted to add, but forgot to mention:

I think this sentence is broken, at least I do not understand it (for `security.warn_submit_secure_to_insecure`):
> If false, no warning is shown when submitting s form from https to http.

Which _“s form”_? Or is it one of this kind if you don’t know it then you don’t will need it (which would be all right from my side)?

And even more nitpicking:
> `FeatureRecommendations` IF false, don't recommend browser features.

IF --> If

**Comment by mkaply** [comment]
> All but one are fine, was this intentional?

I moved one but not the other :)

Thanks for these, I'll get them fixed.

**Comment by mkaply** [comment]
Fixed. It was supposed to be "submitting a form"
